t/t9210-scalar.sh

```diff
@@ -247,6 +247,29 @@ test_expect_success 'scalar reconfigure --all with detached HEADs' '
 	done
 '
 
+test_expect_success 'verify http.<url>.version=HTTP/1.1 for ADO URLs' '
+	test_when_finished rm -rf test-http-url-config &&
+
+	# Create a test repository
+	git init test-http-url-config &&
+
+	# Test both URL types in a loop
+	for url in "https://dev.azure.com/test/project/_git/repo" \
```

Comment:
Modify this URL to be of the form `https://test@dev.azure.com/test/project/_git/repo` to confirm that the recommended URL form the Azure Repos clone UI will work.